SDL/Event.cuh

```diff
@@ -28,11 +28,7 @@
 #include "Constants.h"
 
 #include "cuda_profiler_api.h"
-#ifdef __CUDACC__
-#define CUDA_G __global__
```

Comment:
Is this not needed here anymore?

Reply:
Looking at the code a bit better, is this removed for the same reason that the alpaka header was removed? Because it is included in the `Hit.cuh` file, which is then included in all of the files that need it?

Reply:
Yes, also because the "CUDA_G" definition is only used in Hit.cuh